app/assets/stylesheets/application.scss

```diff
@@ -1,7 +1,7 @@
 $govuk-assets-path: "/register-charity-hmrc/assets/lib/govuk-frontend/govuk/assets/";
```

Comment:
Please add a path to dist i.e. $govuk-assets-path: "/register-charity-hmrc/assets/lib/govuk-frontend/**dist**/govuk/assets/";
 as there are 404s on fonts. See screenshot below.

<img width="1791" alt="test" src="https://github.com/hmrc/charities-registration-frontend/assets/63714619/2a4dd04b-8d34-4143-ac8f-453682d9a76c">